packages/odyssey/src/scss/components/_infobox.scss

```diff
@@ -62,10 +63,6 @@
   justify-content: start;
 }
 
-.is-ods-infobox-info {
-  background-color: cv('blue', '000');
-}
-
```

Comment:
why the CSS changes?

Reply:
If `variant` is not provided, it will not have any default styling applied. Given that info is the default visual appearance of the Infobox, I made it so in the CSS (no class required)